### PR TITLE
refactor(py-sdk)!: clean up validators and bots

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

Drastically simplify the experience of writing ApePay bots, as well as improve how to work with validators as well

Lastly, did some reorganization of the module structure

### How I did it

I wanted to make it easy to work with ApePay _without_ requiring silveback to be installed, but also make it very easy to integrate with

### How to verify it

check out `bots/example.py` and `scripts/demo.py`

### Checklist

- [x] All changes are completed
- [x] New test cases have been added
- [x] Documentation has been updated

### CODE REVIEW

File: bots/example.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from datetime import timedelta
+
+from ape_ethereum import multicall
+from silverback import SilverbackApp
+
+from apepay import StreamManager
+
+app = SilverbackApp()
+# NOTE: You should use one bot per-supported network
+# NOTE: This bot assumes you use a new bot per deployment
```
Comment (mikeshultz) [inline bots/example.py:11]: suggestion: What is this noting? Why does this matter? Might be worth expanding on this if it's important for a user to be aware of.

File: bots/example.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+sm = StreamManager(os.environ["APEPAY_CONTRACT_ADDRESS"])
+
+# NOTE: You would probably want to index this by network and deployment,
+#       if you were operating on multiple networks or deployments
```
Comment (mikeshultz) [inline bots/example.py:15]: question: You can do that with Silverback?

File: bots/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Silverback Example App
+
+These apps are intended to be examples of how to work with your own deployment of the ApePay
+protocol interacting your own infrastructure for a SaaS product, using Silverback for real-time
+events. You can host these examples on the Silverback Cloud (https://silverback.apeworx.io) or
```
Comment (fubuloubu) [inline bots/README.md:5]:
```suggestion
events. You can host these examples on the Silverback Platform (https://silverback.apeworx.io) or
```

File: bots/example.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import os
+from datetime import timedelta
+
+from ape_ethereum import multicall
```
Comment (fubuloubu) [inline bots/example.py:4]: not technically used yet, I had this in some of my other code for making claims I didn't publish yet

**Review by mikeshultz — APPROVED** [review]

File: bots/example.py (hunk 2 above)
Comment (fubuloubu) [reply bots/example.py:15]: This note is a shoutout to the database, that you would probably want to add a column for your network and deployment address for your list of streams

**Pull request merged**